libraries/Microsoft.Bot.Builder.Ai/LocaleConverterMiddleware.cs

```diff
@@ -13,10 +13,11 @@ namespace Microsoft.Bot.Builder.Ai
     /// </summary>
     public class LocaleConverterMiddleware : IMiddleware
     {
-        private ILocaleConverter localeConverter; 
-        private readonly string toLocale;
+        private ILocaleConverter _localeConverter; 
```

Comment:
This seems like it should be readonly, as it's only set in the constructor.

Reply:
updated to be readonly.